pkg/vm/cli/cli.go

```diff
@@ -201,15 +320,31 @@ func isMethodArg(s string) bool {
 func parseArgs(args []string) ([]vm.StackItem, error) {
 	items := make([]vm.StackItem, len(args))
 	for i, arg := range args {
+		var typ, value string
 		typeAndVal := strings.Split(arg, ":")
 		if len(typeAndVal) < 2 {
-			return nil, errors.New("arguments need to be specified as <typ:val>")
+			if typeAndVal[0] == "false" || typeAndVal[0] == "true" {
+				typ = "bool"
```

Comment:
what do you thing about using constants for "bool";"int","string"?

Reply:
Will do.

Reply:
Not sure it actually improved anything, but looks... interesting.